Source/JavaScriptCore/inspector/scripts/codegen/cpp_generator_templates.py

```diff
@@ -189,10 +189,15 @@ class AlternateBackendDispatcher {
 """${classAndExportMacro} ${domainName}FrontendDispatcher {
     WTF_DEPRECATED_MAKE_FAST_ALLOCATED(${domainName}FrontendDispatcher);
 public:
-    ${domainName}FrontendDispatcher(FrontendRouter& frontendRouter) : m_frontendRouter(frontendRouter) { }
+    ${domainName}FrontendDispatcher(FrontendRouter&);
+    ~${domainName}FrontendDispatcher();
+    ${domainName}FrontendDispatcher(const ${domainName}FrontendDispatcher&) = delete;
+    ${domainName}FrontendDispatcher(${domainName}FrontendDispatcher&&) = delete;
+    ${domainName}FrontendDispatcher& operator=(const ${domainName}FrontendDispatcher&) = delete;
+    ${domainName}FrontendDispatcher& operator=(${domainName}FrontendDispatcher&&) = delete;
```

Comment:
could/should we use `WTF_MAKE_NONCOPYABLE` and `WTF_MAKE_NONMOVABLE` instead?

Reply:
It is as many lines (with the header includes) but OK.